ironfish-cli/src/commands/status.ts

```diff
@@ -108,9 +108,9 @@ function renderStatus(content: GetNodeStatusResponse, debugOutput: boolean): str
     content.peerNetwork.peers
   }`
 
-  const blockchainStatus = `${content.blockchain.synced ? 'SYNCED' : 'NOT SYNCED'} @ HEAD ${
-    content.blockchain.head
-  }`
+  const blockchainStatus = `HEAD ${content.blockchain.head}, behind: ${TimeUtils.renderSpan(
```

Comment:
I also don't know exactly how this will answer any user's questions exactly but I'm willing to experiment to see what will happen. I imagine these will come up

 - Users will file bug reports because status always says the word "behind" even when it's fully caught up.
 - It says I'm behind 5 hours but it says I'm synced

Reply:
Yea I thought of those questions coming up as well. Especially the wording of `behind` but didn't think of anything better. One solution would just be only add `SYNCED` `NOT SYNCED` when using `--all`. I think HEAD is still fine because that hash and  behind time are describing the `HEAD` block

Reply:
Ok updated this a bit to be slightly less confusing